src/DataStore.ts

```diff
@@ -1,23 +1,32 @@
-import { GatewayDispatchEvents, GatewayOPCodes, GatewayVoiceServerUpdateDispatch, GatewayVoiceStateUpdateDispatch } from 'discord-api-types/v8/gateway';
-import { Client, Constants, Guild } from 'discord.js';
+import { GatewayOPCodes } from 'discord-api-types/v8/gateway';
+import { GatewayVoiceState } from 'discord-api-types/v8/payloads/voice';
+import { Client, Guild } from 'discord.js';
```

Comment:
We actually export a `WSEvent` object as part of `Constants`.
```suggestion
import { Client, Constants, Guild } from 'discord.js';
```